<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CodeSmith</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>0</y>
      <width>241</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label{
	border-image: url(:/images/images/熊猫2.jpg);
	border-top-left-radius:30px;
	border-bottom-left-radius:30px;
}
#label:hover{
	border-image: url(:/images/images/熊猫.jpg);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>0</y>
      <width>301</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-top-right-radius:30px;
border-bottom-right-radius:30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>286</x>
      <y>30</y>
      <width>174</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_login_view">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_login_view{
	border:none;
}
#btn_login_view:hover{
	padding-top:5px;
	padding-left:5px;
}
#btn_login_view:focus{
	color: rgb(186, 186, 186);
}</string>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cfg_view">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_cfg_view{
	border:none;
}
#btn_cfg_view:hover{
	padding-top:5px;
	padding-left:5px;
}
#btn_cfg_view:focus{
	color: rgb(186, 186, 186);
}</string>
       </property>
       <property name="text">
        <string>配置</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_login" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>90</y>
      <width>271</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLineEdit" name="let_uname">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#let_uname{
	border:1px solid rgb(0,0,0);
	border-radius:8px;
}
#let_uname:hover{
	border: 2px solid blue; /* 悬停时的边框颜色 */
	background-color: rgba(189, 255, 177, 0.1); /* 悬停时的背景颜色，半透明 */
}
#let_uname:focus {
     border: 2px solid white; /* 获得焦点时的边框颜色 */
}


</string>
     </property>
     <property name="placeholderText">
      <string>账号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="let_pwd">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#let_pwd{
	border:1px solid rgb(0,0,0);
	border-radius:8px;
}
#let_pwd:hover{
	border: 2px solid blue; /* 悬停时的边框颜色 */
	
     background-color: rgba(189, 255, 177, 0.1); /* 悬停时的背景颜色，半透明蓝色 */
}
#let_pwd:focus {
     border: 2px solid white; /* 获得焦点时的边框颜色 */
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#btn_login{
	background-color: rgb(118, 118, 118);
	color: rgb(255, 255, 255);
	border:1px solid rgb(0,0,0);
	border-radius: 8px;
}
#btn_login:hover{
	background-color:rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}
#btn_login:pressed{
	padding-top:5px;
	padding-left:5px;
}
</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_info">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lab_uname">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>16</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/resource.qrc">:/icons/icons/用户名.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lab_pwd">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>64</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/resource.qrc">:/icons/icons/密码.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear">
     <property name="geometry">
      <rect>
       <x>229</x>
       <y>16</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#btn_clear{
	border:none;
}
#btn_clear:hover{
	padding-top:3px;
	padding-left:3px;
}</string>
     </property>
     <property name="icon">
      <iconset resource="../res/resource.qrc">
       <normaloff>:/icons/icons/清空.png</normaloff>:/icons/icons/清空.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_show">
     <property name="geometry">
      <rect>
       <x>232</x>
       <y>67</y>
       <width>13</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#btn_show{
	border:none;
}
#btn_show:hover{
	padding-top:3px;
	padding-left:3px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res/resource.qrc">
       <normaloff>:/icons/icons/密码显示.png</normaloff>:/icons/icons/密码显示.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_close{
	border:none;
}
#btn_close:hover{
	padding-top:5px;
	padding-left:5px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../res/resource.qrc">
      <normaloff>:/icons/icons/关闭.png</normaloff>:/icons/icons/关闭.png</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="widget_cfg" native="true">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>90</y>
      <width>271</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="btn_cert">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#btn_cert{
	border:2px solid rgb(0,0,0);
	border-radius: 5px;
}

#btn_cert:hover{
	padding-top:5px;
	padding-left:5px;
}
</string>
     </property>
     <property name="text">
      <string>请选择数字证书存放位置</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_cert">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>widget_cfg</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>widget</zorder>
   <zorder>widget_login</zorder>
   <zorder>btn_close</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../res/resource.qrc"/>
 </resources>
 <connections/>
</ui>
